example/src/index.tsx

```diff
@@ -66,6 +68,7 @@ export default function PaperExample() {
   const [isDarkMode, setIsDarkMode] = React.useState(false);
   const [themeVersion, setThemeVersion] = React.useState<2 | 3>(3);
   const [rtl, setRtl] = React.useState<boolean>(I18nManager.isRTL);
+  const [collapsed, setCollapsed] = React.useState<boolean>(false);
```

Comment:
Passing the type can be skipped when declaring useState when dealing with primitive types

```suggestion
  const [collapsed, setCollapsed] = React.useState(false);
```